common/app/model/Element.scala

```diff
@@ -94,6 +94,11 @@ final case class VideoMedia(videoAssets: List[VideoAsset]) {
     }.flatten.sorted
   }
   val duration: Int = videoAssets.headOption.map(_.duration).getOrElse(0)
+  val formattedDuration: String = {
+    videoAssets.headOption.map(_.durationMinutes).getOrElse(0).toString() + ":" + "%02d".format(videoAssets.headOption.map(_.durationSeconds).getOrElse(0))
+  }
+  val durationMinutes: Int = videoAssets.headOption.map(_.durationMinutes).getOrElse(0)
```

Comment:
Should these not be options, as, if the time isn't there, it should be `None` not 0?